### PR TITLE
add comment doc: FieldNameTuple and Fields are always in-sync.

### PR DESCRIPTION
**Pull request by mw66**
@radcapricorn

### CODE REVIEW

**Comment by mingwugmail** [comment]
The point of doc is that the user don't have to dive into the code to learn its usage.

**Comment by atilaneves** [comment]
Personally I think these comments are more confusing than their absence. Maybe it's just me, but why wouldn't they be in the same order and have the same length? It's strictly dependent on the order of field declaration.

**Comment by mingwugmail** [comment]
Then given a tuple of field names how do the user know their corresponding types? e.g.

https://github.com/mingwugmail/jdiutil/blob/master/source/jdiutil.d#L105

how do I know types[i] is correctly corresponding to fields[i]?

You are an language expert, so you know. But to a normal user, esp new-comer to the D lang, how would s/he know by reading the doc (without dig into phobos source code)?

https://forum.dlang.org/thread/xipkvmafkradzyygcgdg@forum.dlang.org

**Comment by n8sh** [comment]
>Personally I think these comments are more confusing than their absence. Maybe it's just me, but why wouldn't they be in the same order and have the same length? It's strictly dependent on the order of field declaration.

Then perhaps a better remark is "Fields appear in order of declaration" instead of a note about being in sync with some other thing. I agree that the remark about number of fields is superfluous.

File: std/experimental/checkedint.d [hunk 1]
```diff
@@ -1083,6 +1083,25 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
...
+}
+
+// toString
+@safe unittest
+{
+    Checked!(int, void) x;
+    x = 42;
+    assert(x.toString == "42");
+    x = -42;
+    assert(x.toString == "-42");
```
Comment (atilaneves) [inline std/experimental/checkedint.d:1104]: How is this related to the PR title?

**Comment by mingwugmail** [comment]
sorry branch messed up.

File: std/experimental/checkedint.d (hunk 1 above)
Comment (mingwugmail) [reply std/experimental/checkedint.d:1104]: sorry branch messed up.

**Comment by RazvanN7** [comment]
@mingwugmail What is the status of this PR? It seems like it needs a rebase.
